### PR TITLE
Add Karol Wojtaszek to elixir devs

### PR DESCRIPTION
**Pull request by kwojtaszek**

### CODE REVIEW

File: elixir-users.geojson [hunk 1]
```diff
@@ -775,6 +775,20 @@
         "twitter": "amatalai",
         "marker-color": "#FFDF5B"
       }
-    }
+    },
+    {
+     "type": "Feature",
+     "geometry": {
+       "type": "Point",
+       "coordinates": [16.902798, 52.455602]
```
Comment (igas) [inline elixir-users.geojson]: pls, swap coordinates. Github have different order.

Comment (kwojtaszek) [inline elixir-users.geojson]: Are you sure? It looks ok on the map

Comment (igas) [inline elixir-users.geojson]: Hm, I can't see it in visual diff in Poland. Ok, if you say so =)

**Comment by igas** [comment]
:heart: :green_heart: :blue_heart: :yellow_heart: :purple_heart:

**Pull request merged**